test/extended/util/framework.go

```diff
@@ -238,10 +238,56 @@ func processScanError(log string) error {
 	return fmt.Errorf(log)
 }
 
+// WaitForImageStreams creates custom ruby imageStream in OpenShift namespace
+// TODO: To eliminate the dependency on OpenShift Samples Operator in future,
+// WaitForImageStreams should be a replacement of WaitForOpenShiftNamespaceImageStreams func
+func WaitForImageStreams(oc *CLI) error {
```

Comment:
We can do better here and add a utility function that wraps `oc import-image` generically.

```suggestion
// ImportImageStream imports the provided image reference to an imagestream with the given name in the
// oc client's current namespace.
func ImportImageStream(oc *CLI, imageRef string, imageStreamName string) error {
```